roles/profiles/logging/meta/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+dependencies:
+  - elasticsearch
+  - kibana
```

Comment:
On the second thought, maybe my kibana PR is implemented incorrectly so that kibana role should ensure running also kibana/proxy. OK, nevermind my previous comment. I will fix my PR.